st2common/st2common/services/workflows.py

```diff
@@ -67,8 +67,20 @@ def request(wf_def, ac_ex_db):
         ac_ex_db.context
     )
 
+    context = {
+        "st2": {
+            "workflow_id": str(ac_ex_db.id), 
+        },
+    }
+
+    context['st2'].update(ac_ex_db.context)
+
     # Instantiate the workflow conductor.
-    conductor = conducting.WorkflowConductor(wf_spec, **action_params)
+    conductor = conducting.WorkflowConductor(
+        wf_spec,
+        inputs=action_params,
+        context=context
+    )
```

Comment:
Just my style preference.  Can we do something like https://github.com/StackStorm/orchestra/blob/master/orchestra/tests/unit/conducting/test_workflow_conductor.py#L76?